Makefile

```diff
@@ -109,6 +109,7 @@ test_cover:
 
 test_lint:
 	gometalinter.v2 --disable-all --enable='golint' --enable='misspell' --enable='unconvert' --enable='ineffassign' --linter='vet:go vet -composites=false:PATH:LINE:MESSAGE' --enable='vet' --deadline=500s --vendor ./...
+	!(gometalinter.v2 --disable-all --enable='errcheck' --vendor ./... | grep -v "client/")
```

Comment:
Reason for the `!` here is that grep returns error code `0` iff it returns any lines, and 1 if it doesn't return any lines.